core_backend/app/llm_call/parse_input.py

```diff
@@ -116,6 +116,48 @@ async def wrapper(
     return wrapper
 
 
```

Comment:
can we put this after the `_identify_language()` function so that can be right after its decorator form for clarity?